setup.py

```diff
@@ -39,6 +39,9 @@ def package_data(pkg, roots):
         'https://github.com/edx/xblock-utils/tarball/c39bf653e4f27fb3798662ef64cde99f57603f79#egg=xblock-utils',
     ],
     entry_points={
+        "lms.djangoapp": [
+            "rg_learning_time = lti_consumer.apps:ApiConfig"
```

Comment:
Why?

Reply:
I've retested it one more time. It's not necessary. 
This part was deleted.